mrp_business/models/product_product.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import re
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    # ========== Trigram
+    PRODUCT_CODE_GENERIC_TLA = fields.Char(default="XXX")
```

Comment:
c'est une constante, si je comprend bien non ? 

```suggestion
    _PRODUCT_CODE_GENERIC_TLA = "XXX
```

Reply:
ah bah oui tiens bizarre ça. Je change

Reply:
Ah oui mais non. Sur le BoM c'est related à ce champ. Donc si je le mets en constante privé, je peux plus facilement aller le chercher

Reply:
ce que je veux dire, c'est que tu ne souhaites pas une colonne en plus dans product_product, si ? tu veux juste avoir accès à une constante.
Donc, glisser qq part un ``PRODUCT_CODE_GENERIC_TLA`` = "XXX" est suffisant.

àprès si tu as besoin d'y accéder ailleurs, tu peux faire un  ``from odoo.addons.mrp_business.models.product_product import PRODUCT_CODE_GENERIC_TLA``